feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package setup is scoped only to be used for scripts in path
+// feature/experimental/system/gnmi/benchmarking/ate_tests/
+// Do not use elsewhere.
+package setup
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	// ISISInstance is ISIS instance name.
+	ISISInstance = "DEFAULT"
+	// PeerGrpName is BGP peer group name.
+	PeerGrpName = "BGP-PEER-GROUP"
```

Comment:
not used outside the package, un-export it

Reply:
This is exported by PR #645, so just left it as it is.

Reply:
Will let these through as is. In the future, better to leave it unexported in the PR that introduces it and change it to exported in the PR that needs it to be.

Reply:
Point noted. Thanks Greg.